http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/LeoAppServiceInterp.scala

```diff
@@ -600,28 +596,14 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
         F.raiseError[Unit](AppAlreadyExistsInWorkspaceException(workspaceId, appName, c.app.status, ctx.traceId))
       )
 
-      // Get the Landing Zone Resources for the app for Azure
-      leoAuth <- samDAO.getLeoAuthToken
-      landingZoneResourcesOpt <- cloudContext.cloudProvider match {
-        case CloudProvider.Gcp => F.pure(None)
-        case CloudProvider.Azure =>
-          for {
-            landingZoneResources <- wsmDao.getLandingZoneResources(workspaceDesc.spendProfile, leoAuth)
-          } yield Some(landingZoneResources)
-      }
-
-      // Get the optional storage container for the workspace
-      storageContainer <- wsmDao.getWorkspaceStorageContainer(workspaceId, userToken)
-
       // Validate the machine config from the request
       // For Azure: we don't support setting a machine type in the request; we use the landing zone configuration instead.
       // For GCP: we support setting optionally a machine type in the request; and use a default value otherwise.
       machineConfig <- (cloudContext.cloudProvider, req.kubernetesRuntimeConfig) match {
         case (CloudProvider.Azure, Some(_)) =>
           F.raiseError(AppMachineConfigNotSupportedException(ctx.traceId))
-        // TODO: pull this from the landing zone instead of hardcoding once TOAZ-232 is implemented
         case (CloudProvider.Azure, None) =>
-          F.pure(KubernetesRuntimeConfig(NumNodes(1), MachineTypeName("Standard_A2_v2"), false))
+          F.pure(KubernetesRuntimeConfig(NumNodes(1), MachineTypeName("unset"), false))
```

Comment:
`Standard_A2_v2` was a lie anyway. :(
The nodepool machine type is defined in the LZ, just marking it as `unset` in the Leo DB.

This is an example where the Leo DB schema is overloaded to support GCP and Azure, which have different models.

Reply:
any value in updating it after the runtime is created in leo DB?

Reply:
So this is actually the AKS nodepool machine type for apps. We could update it in back Leo, but it would just be informational, since we don't rely on this field for AKS apps.

Reply:
do we display cost for Azure? wonder if this info will be needed for cost display

Reply:
I confirmed the cost appears in the UI -- I think it just displays runtime cost though, not app costs. Does the UI report on app (e.g. Cromwell, WDS) app costs?

Reply:
Yeah I think prod doesn't display app costs in the UI. E.g. here's a workspace with 2 apps and 0 runtimes:
<img width="121" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/5368863/81f02e31-e0d3-4ccb-8bbc-788f029b1064">

So we could correctly set the node type + region for apps, but I'm sort of inclined to leave that for a separate PR if they are unused today.

Reply:
For completeness: my BEE displaying costs for runtimes :) 
<img width="106" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/5368863/5d6d2f46-47bd-4e43-8355-bce3a074f5ca">

Reply:
@LizBaldo pointed out to me that the app region is displayed in the Cloud Environments page. Here is my BEE:
<img width="621" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/5368863/afbb95e1-25d9-4950-9801-508a4db193d2">

Here is prod: 
<img width="626" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/5368863/f4384ea3-9d10-4cad-a52f-1a7e281fe3cf">

Incidentally -- prod is incorrect, `us-central1` is a GCP region. I'll change it to fill in the correct region and machine type for apps as well.

Reply:
Update: fixed with my latest commit. In my BEE:

<img width="632" alt="image" src="https://github.com/DataBiosphere/leonardo/assets/5368863/90402946-ca31-45c5-8359-1286ae6356af">